index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    <title>Youtube</title>
+</head>
+<body>
+    <nav class="navbar navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand " href="#">
+            <img class="logo"  src="images/youtube_logo.PNG" alt="" width="110" height="51" class="d-inline-block align-text-top">
+          </a>
+        </div>
+      </nav>
+
+    <div class="container">
```

Comment:
- [ ] Please🙏 kindly replace the highlighted `div` tag with a `main` tag. This is considered to be good 👍 practice in HTML semantics, as the main content should be wrapped in a `main` tag.

Reply:
- [ ] Nice work explaining the improvements to the student. However, Instead of giving orders, you could make suggestions.
You could say for example :
Good job creating a container 👍, however, It could be better to use `main` tag instead of `div`.  Kindly fix it🤝.
What do you think about refreshing yourself about [professional communication in a code review?](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) :🤗And then update your comment in a polite and descriptive wy